docs/source/InputsOutputs.rst

```diff
@@ -0,0 +1,303 @@
+
+   * - Filename
+     - Description
+   * - glorys_ic_1993-01-01.nc
+     - ocean physical initial conditions from Glorys (tools/initial/write_glorys_initial.py)   
+   * - NWA12_COBALT_2023_10_spinup_2003.nc
+     - ocean biogeochemical initial conditions (tools/initial/write_bgc_initial.py)    
+   * - so_*.nc, thetao_*.nc, zos_*.nc, uv_*.nc 
+     - Open boundary conditions for physical variables (tools/boundary/write_glorys_boundary.py)   
+   * - bgc_esper_1993_2022.nc
+     - Open boundary conditions for ALK and DIC (tools/boundary/write_bgc_boundary.py; Prerequisite: tools/boundary/esper/run_esper.sh)    
+   * - ERA5_*.nc
+     - atmos forcings from ERA5 (tools/atmos)
+   * - glofas_runoff_****.nc
+     - River runoff from GloFAS data (tools/rivers/write_runoff_glofas.py)    
+   * - RiverNutrients_Integrated_NWA12_GLOFAS_RC4US1990to2022_2023_04_v2.nc
+     - Biogeochemical tracer concentrations for river runoff 
+   * - mole_fraction_of_co2_extended_ssp245.nc
+     - mole fraction of carbon dioxide in atmos
+
+
+.. _model_configureFile:
+
+---------------------------
+Model_configure files
+---------------------------
+
+The model configurstion files for regional MOM6-SIS2-COBALT configurations are listed and described in :numref:`Table %s <ModelConfig>`.
+
+.. _ModelConfig:
+
+.. list-table:: *Model configuration files*
+   :widths: 40 50
+   :header-rows: 1
+
+   * - Filename
+     - Description
+   * - input.nml
+     - Fortran namelist file containing parameters that control model run.
+   * - MOM_input
+     - This input file provides the adjustable run-time parameters for MOM6
+   * - MOM_override
+     - Override MOM6 run-time parameters 
+   * - MOM_layout
+     - Control MOM6 model's layout
+   * - SIS_input
+     - This input file provides the adjustable run-time parameters for SIS2
+   * - SIS_override
+     - Override SIS2 run-time parameters 
+   * - SIS_layout
+     - Control SIS2 model's layout
+   * - field_table
+     - An ASCII table that is used to register tracer fields
+   * - diag_table
+     - An ASCII table that is used to control model outputs
+   * - data_table
+     - An ASCII file that is used to control external data forcing fields, such as surface forcings or river runoff
+
+The data_table is commonly formatted by specifying each of the fields in the order listed below, with a new line for each entry.
+
+| ``gridname``: The component of the model this data applies to. eg. `atm` `ocn` `lnd` `ice`.
+| ``fieldname_code``: The field name according to the model component. eg. `salt`
+| ``fieldname_file``: The name of the field within the source file.
+| ``file_name``: Path to the source file.
+| ``interpol_method``: Interpolation method eg. `bilinear`
+| ``factor``: A scalar by which to multiply the field ahead of passing it onto the model. This is a quick way to do unit conversions for example.
+
+Example Format:
+
+.. code-block:: console
+
+   "ATM", "t_bot",  "t2m", "./INPUT/2t_ERA5.nc", "bilinear", 1.0
+
```

Comment:
We could also mention that you can set a constant value by entering empty quotes for `fieldname_file` and `file_name` and setting `interpol_method` to `"none"`. For example, for a constant atmospheric oxygen value,

`"ATM", "o2_flux_pcair_atm",  "",       "",                           "none",      0.214`